src/components/Book.jsx

```diff
@@ -1,12 +1,16 @@
 import './Book.css';
+import { useEffect } from 'react';
 import { CircularProgressbar } from 'react-circular-progressbar';
 import 'react-circular-progressbar/dist/styles.css';
 import { useDispatch, useSelector } from 'react-redux';
-import { removeBook } from '../features/books/bookSlice';
+import { getBook, removeBook } from '../features/books/bookSlice';
 
 const Book = () => {
-  const books = useSelector((store) => store.book);
+  const books = useSelector((store) => store.book.books);
   const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBook());
+  }, [dispatch, books]);
 
   return books.map((book) => (
```

Comment:
- [ ] [OPTIONAL] I noticed that there is no loader to indicate that the action is pending. This could lead to confusion for the user, who may not know whether the action has been carried out or not (despite the `success` message). I would recommend adding a loader to improve the user experience.